### ISSUE TITLE
UI Suggestion: Move a red dot which indicates non-matched relations to the right after the word placement.

### ISSUE THREAD

**Issue by gulivan** [azimuttapp/azimutt#284]
Thank you for an amazing tool!

Please consider aligning the red dot to the right after the word because in the current implementation, this dot blocks the last letter.

<img width="496" alt="image" src="https://github.com/azimuttapp/azimutt/assets/34831957/53597ee2-0c7f-40df-8671-064a39da4dac">

How it could look like:

<img width="593" alt="image" src="https://github.com/azimuttapp/azimutt/assets/34831957/4a1ff1ac-bf77-4dce-a586-6a724f2bec01">

<img width="300" alt="image" src="https://github.com/azimuttapp/azimutt/assets/34831957/8eaac7dc-4985-4490-bc98-45f485adf73c">

**Comment by loicknuchel** [comment]
Indeed, or maybe aligning on the left to avoid conflict with SQL comment like in your screen.
Looking into this ;)
